CONTRIBUTING.md

```diff
@@ -227,7 +227,7 @@ tox -e py312-test-instrumentation-aiopg
 Some of the tox targets install packages from the [OpenTelemetry Python Core Repository](https://github.com/open-telemetry/opentelemetry-python) via pip. The version of the packages installed defaults to the main branch in that repository when tox is run locally. It is possible to install packages tagged with a specific git commit hash by setting an environment variable before running tox as per the following example:
 
 ```sh
-CORE_REPO_SHA=c49ad57bfe35cfc69bfa863d74058ca9bec55fc3 tox
+CORE_REPO_SHA=966750aad4ba2a9c123b4f14785958ffe50b54ae tox
```

Comment:
I'm not sure we want to keep this up to date :sweat_smile:

Reply:
I need a few features from the not-yet-released core (semconv 1.28.0 and event SDK for tests). I think I can work around both needs by hardcoding attribute names and implementing custom event SDK in tests.

but out of curiosity, how do we normally handle things like this - wait for the core repo to release?

A related question - it seems we last released core two months ago - is a new release planned any time soon or should I do the workarounds?

Reply:
By default it should use main so you won't need to touch anything usually

Reply:
> but out of curiosity, how do we normally handle things like this - wait for the core repo to release?

The SHA points to a commit in the core repo (usually unreleased when you change it) to pull dev changes from the other repository that is needed for the current PR. As @xrmx has mentioned, USUALLY you can set it to `main` to pick up the latest changes from core. In the off chance that you make two simultaneous PRs in both core and contrib that need changes from each other, this is when we usually set the SHA to specific commits of the respective PRs.

> A related question - it seems we last released core two months ago - is a new release planned any time soon or should I do the workarounds?

We will probably do a release soon once this gets merged.

Reply:
> We will probably do a release soon once this gets merged.

We can release earlier since this instrumentation will be released separately on its own